### DISCUSSION TITLE
240p compatibility?

### DISCUSSION THREAD

**Discussion by BuckoA511** [djyt/cannonball#129] [Q&A]
So I know this is pretty niche but... I'm trying to run Cannonball on a CRT monitor at Outrun's native 320x240p. I've installed the drivers for the PC (CRT Emudriver) and they are working.

I start Cannonball in 240p and it initially works, then after anything from a few minutes to a few seconds it starts to glitch, first a couple of flickering lines which get worse until the whole screen is just a garbled mess of graphics.

It's a long shot but is there anything I can do to get the game to work at Outrun's native resolution?

**Comment by glopgar** [comment]
Hi, 
I know this is not the answer to your question but if you own a Wii, there is a Cannonball version that works great at 240p.
http://www.wiibrew.org/wiki/OutRun_(Cannonball_engine)
